### PR TITLE
Add csv to filter

### PR DESCRIPTION
**Pull request by nikola-rados**
## Overview

We have some `csv` files we want to make available on `THREDDS`. 

## Changes

**Non-breaking changes**
- Adds extra file type to `catalog.xml` filter

**Breaking changes**
- None

## Related Issue / Discussion

- None (although I could make one then edit this)

## Additional Information

This is a very small change, and I wasn't sure it was worth it to create (and maintain) a fork just for this. Furthermore, I couldn't really find a good way to make this more dynamic without adding a bunch of unnecessary complexity.

### CODE REVIEW

**Review by huard — APPROVED** [review]

**Review by tlvu — APPROVED** [review]
LGTM.  Next time, feel free to create a branch directly from this birdhouse-deploy repo, you have write access.

We recently changed our release procedure.  There is now a CHANGE.md file and bumpversion procedure as well.

I was going to go this for you since this is your first time with this new procedure but I do not  have access to your fork.

So basically what you need to do before merging this PR is
* update that CHANGE.md
* run `bump2version patch`
* push the changes
* merge this PR

Sample other PR that started following this new procedure https://github.com/bird-house/birdhouse-deploy/pull/180

**Comment by tlvu** [comment]
> This is a very small change, and I wasn't sure it was worth it to create (and maintain) a fork just for this. 

Absolutely, this is a small non-disruptive change, don't need a fork for this.

> Furthermore, I couldn't really find a good way to make this more dynamic without adding a bunch of unnecessary complexity.

FYI, you are free to make disruptive very specific to PCIC change, just use the override mechanism.  Ouranos have a repo with override for our very specific needs that are not generic enough to keep them in this birdhouse-deploy generic repo.  See https://github.com/bird-house/birdhouse-deploy-ouranos

**Comment by tlvu** [comment]
@nikola-rados did you force push to remove the CHANGE.md update and the bumpversion?  Did you encounter any problems?  If you give me access to your nikola-rados/birdhouse-deploy fork I can perform those release steps for you then you can merge the PR.

Sorry it was never been meant to let you on your own the first time we rollout a new process.  Let me know.

**Comment by nikola-rados** [comment]
@tlvu I did encounter some issues. I wasn't sure if I had done things correctly since the `Labeler` job failed. I have since redone the process and it has still failed. What exactly is this item checking? I've added you as a collaborator on the repo.

**Comment by tlvu** [comment]
> I did encounter some issues. I wasn't sure if I had done things correctly since the `Labeler` job failed. I have since redone the process and it has still failed.

@fmigneault labeler job log https://github.com/bird-house/birdhouse-deploy/runs/3058569185

```
Run actions/labeler@v2
  with:
    repo-token: ***
    configuration-path: .github/labeler.yml
    sync-labels: false
Error: HttpError: Resource not accessible by integration
Error: Resource not accessible by integration
```

Could this job fail simply because the source branch of the PR is from a fork?

**Comment by tlvu** [comment]
@nikola-rados Everything seems looks good.  I've merged this PR for you, sorry for all the trouble for such little change.  Feel free to create a branch directly in this repo next time.  Others will also be able to easier assist you if anything.

**Comment by nikola-rados** [comment]
@tlvu no problem, I will create a more direct branch next time!

**Pull request merged**
